targets/xorg

```diff
@@ -108,6 +108,8 @@ install xorg arch=,xserver-xorg-video-fbdev$ltspackages
 if [ "${ARCH#arm}" = "$ARCH" ]; then
     install arch=,xserver-xorg-video-intel$ltspackages
 fi
+# JMT: eventually wrap this removal with an appropriate test
+remove xf86-video-odroid-c1
```

Comment:
To install only necessary packages, we need two things:  one, a list of universally required packages; and two, a way to identify individual devices or features.  Your suggestion for the first (`xorg-server`, `xorg-server-utils` and `xorg-apps`) looks like a great start.  I have no idea how to do the second.  Is there an elegant way to check whether a device is an odroid, for instance?